pkgs/data/themes/alacritty-theme/default.nix

```diff
@@ -1,18 +1,18 @@
 { lib
 , fetchFromGitHub
-, nix-update-script
+, unstableGitUpdater
 , stdenvNoCC
 , ... }:
 
 stdenvNoCC.mkDerivation (self: {
   name = "alacritty-theme";
```

Comment:
strictly speaking unrelated to this PR, but any reason that this uses `name` instead of `pname`?